packages/mechanic/app/pages/Nav.js

```diff
@@ -1,31 +1,46 @@
 import React from "react";
+import Case from "case";
 import { useHistory } from "react-router-dom";
+import { DropdownIcon } from "@designsystemsinternational/mechanic-ui-components";
 import * as css from "./Nav.module.css";
 
-export const Nav = ({ name, functionsNames }) => {
+export const Nav = ({ name, functions }) => {
   let history = useHistory();
-
+  const functionsNames = Object.keys(functions);
+  console.log(functions);
+  const currentFn = functions[name];
+  const currentLabel = currentFn.settings.name || Case.title(name);
```

Comment:
There is a `name` setting also now? What are the rules of this new setting?

Reply:
adds the option to name the function some thing different than the key. designers will appreciate this